### PR TITLE
1. updated path for event snapshot/s 2. Add delete snapshots call

### PR DESCRIPTION
**Pull request by jenjenp**

### CODE REVIEW

File: services.yaml [hunk 1]
```diff
@@ -1791,7 +1791,7 @@ paths:
           $ref: "#/definitions/UnauthorizedError"
         404:
           description: NOT FOUND
-  /services/{service_id}/events/${event_id}/snapshot:
```
Comment (chook) [inline services.yaml]: didn't we say we'll leave this as last-snapshot?

File: services.yaml [hunk 2]
```diff
@@ -1847,6 +1847,18 @@ paths:
...
         404:
           $ref: "#/definitions/NotFoundError"
+    delete:
+      tags:
+        - Events
+      summary: "Delete all event snapshots"
+      description: "Removes all existing snapshots for an event. All snapshots will be permanently deleted. Event statistics will be kept."
+      responses:
+        200:
+          description: OK
```
Comment (chook) [inline services.yaml]: please use the #/responses

File: services.yaml (hunk 1 above)
Comment (jenjenp) [reply services.yaml]: fixed

File: services.yaml (hunk 2 above)
Comment (jenjenp) [reply services.yaml]: fixed

File: services.yaml [hunk 3]
```diff
@@ -2124,12 +2124,12 @@ paths:
           $ref: "#/responses/UnauthorizedError"
         404:
           $ref: "#/responses/NotFoundError"
-  /services/{service_id}/events/{event_id}/snapshot:
+  /services/{service_id}/events/{event_id}/last-snapshot:
```
Comment (chook) [inline services.yaml]: was someone ever tasked with making this change?

**Pull request merged**